content/learning-paths/smartphones-and-mobile/debugging_with_mte_on_pixel8/_review.md

```diff
@@ -0,0 +1,60 @@
+---
+review:
+    - questions:
+        question: >
+             Which of the below statements is true?
+        answers:
+            - Memory safety bugs are errors in handling memory by the hardware. Although important, they are not very common in the Android codebase.
+            - Memory safety bugs are errors in handling memory by the software. Although important, they are not very common in the Android codebase.
+            - Memory safety bugs are errors in handling memory by the software. They are important and very common in the Android codebase.
+            
+        correct_answer: 3                    
+        explanation: >
+            Memory safety bugs are errors in handling memory by software. Memory safety bugs are the most common issue in the Android codebases. They account for over 70% of high severity security vulnerabilities and for millions of user-visible crashes.
+
+    - questions:
+        question: >
+           Which of the below statements is false?
+        answers:
+            - The MTE Test application implements common memory safety bugs.
+            - The MTE Test application implements four common memory safety bugs.
+            - The MTE Test application implements four function that trigger memory safety bugs.
+            - The MTE Test application implements four functions which are common in the Android codebase.
+        correct_answer: 4                   
+        explanation: >
+            The MTE Test application implements common memory safety bugs. It implements four common memory safety bugs using four functions. Each funtion triggers a memory safety bug.
+               
+    - questions:
+        question: >
+            Which of the below statements is true?
+        answers:
+            - To debug an application in Android Studio with MTE you need to enable MTE in the phone settings.
```

Comment:
the wrong answers are imprecise - they're technically correct, they're just not sufficient.
Better to word them with an "only" to make them definitely wrong:
- To debug an application in Android Studio with MTE you only need to enable MTE in the phone settings.
- To debug an application in Android Studio with MTE you only need to enable MTE in the Android manifest.
- To debug an application in Android Studio with MTE you only need to launch Android Studio with the device connected.